examples/tensorflow/train.py

```diff
@@ -0,0 +1,173 @@
+# tensorflow 2.x core api
+import logging
+from urllib.parse import urlparse
```

Comment:
Let's remove this. It's not used.

Reply:
Yeah, urllib was used earlier when I had used url tracking type, now its not needed